src/core_plugins/input_control_vis/public/control/filter_manager/range_filter_manager.js

```diff
@@ -76,7 +76,10 @@ export class RangeFilterManager extends FilterManager {
       range = _.get(kbnFilters[0], ['range', this.fieldName]);
     }
 
-    return fromRange(range);
+    if (range == null) {
```

Comment:
Just double-checking that you wanted a `==` instead of a `===`. Might want to slap a comment in here, if you do, as this is usually frowned upon, but it is a handy way to check for null and undefined simultaneously.